docs/source/conf.py

```diff
@@ -98,7 +98,7 @@
 copyright = u'2014-2015, Red Hat'
 
 version = VERSION
-release = 0
+release = '0'
```

Comment:
That is weird... probably related to sphinx versions... on my system:

```
$ python -m unittest discover selftests.doc 
.
----------------------------------------------------------------------
Ran 1 test in 14.878s

OK
```

Anyway, the fix looks good. Thanks!